src/sql/workbench/services/notebook/common/localContentManager.ts

```diff
@@ -25,8 +24,12 @@ export class LocalContentManager implements nb.ContentManager {
 	constructor(@IFileService private readonly fileService: IFileService) { }
 
 	public async loadFromContentString(contentString: string): Promise<nb.INotebookContents> {
-		let contents: JSONObject = json.parse(contentString);
-
+		let contents: JSONObject;
```

Comment:
looks like we could wrap this into a method since it is duplicated twice in this PR.